dotnet-client-libraries/FluentEmail/FluentEmailExample/Controllers/EmailController.cs

```diff
@@ -0,0 +1,124 @@
+        {
+            EmailMetadata emailMetadata = new("john.doe@gmail.com"
+                , "FluentEmail Test email"
+                , "This is a test email from FluentEmail.");
+
+            await _emailService.Send(
+                emailMetadata);
+
+            return Ok();
+        }
+
+        [HttpGet("razortemplate")]
+        public async Task<IActionResult> SendEmailWithRazorTemplate()
+        {
+            User model = new("John Doe", "john.doe@gmail.com", "Platinum");
+
+            EmailMetadata emailMetadata = new(model.Email
+                , "FluentEmail test email with razor template");
+
+            var template = "Dear <b>@Model.Name</b>, </br>" +
+            "Thank you for being an esteemed <b>@Model.MemberType</b> member.";
+
+            await _emailService.SendUsingTemplate(
+                emailMetadata, template, model);
+
+            return Ok();
+        }
+
+        [HttpGet("liquidtemplate")]
+        public async Task<IActionResult> SendEmailWithLiquidTemplate()
+        {
+            User model = new("Jane Doe", "jane.doe@gmail.com", "Gold");
+
+            EmailMetadata emailMetadata = new(model.Email
+                , "FluentEmail test email with liquid template");
+
+            var template = @"Dear <b>{{ Name }}</b>,</br>
+            Thank you for being an esteemed <b>{{ MemberType }}</b> member.";
+
+            await _emailService.SendUsingTemplate(
+                emailMetadata, template, model);
+
+            return Ok();
+        }
+
+
+        [HttpGet("razortemplatefromfile")]
+        public async Task<IActionResult> SendEmailWithRazorTemplateFromFile()
+        {
+            User model = new("John Doe", "john.doe@gmail.com", "Platinum");
+
+            EmailMetadata emailMetadata = new(model.Email
+                , "FluentEmail test email with razor template file");
+
+            string templateFile = $"{Directory.GetCurrentDirectory()}/Mytemplate.cshtml";
+
+            await _emailService.SendUsingTemplateFromFile(
+                emailMetadata, templateFile, model);
+
+            return Ok();
+        }
+
+        [HttpGet("withattachment")]
+        public async Task<IActionResult> SendEmailWithAttachment()
+        {
+            EmailMetadata emailMetadata = new("john.doe@gmail.com"
+                , "FluentEmail Test email"
+                , "This is a test email from FluentEmail."
+                , $"{Directory.GetCurrentDirectory()}/Test.txt");
+
+            await _emailService.SendWithAttachment(
+                emailMetadata);
+
+            return Ok();
+        }
+
+        [HttpGet("multipleemail")]
+        public async Task<IActionResult> SendMultipleEmails()
+        {
+            List<User> users = new();
+
+            User user1 = new("John Doe", "john.doe@gmail.com", "Platinum");
+            users.Add(user1);
+
+            User user2 = new("Jane Doe", "jane.doe@gmail.com", "Gold");
+            users.Add(user2);
```

Comment:
We can define these users in the object initializer for the list instead of calling the `Add()` method.